charts/fybrik/templates/manager-auth-proxy-rb.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if include "fybrik.isEnabled" (tuple .Values.manager.enabled (or .Values.coordinator.enabled .Values.worker.enabled)) }}
+{{- if not (.Values.clusterScoped) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "fybrik.fullname" . }}-proxy-rb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ template "fybrik.fullname" . }}-proxy-role
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.manager.serviceAccount.name | default "default" }}
+  namespace: {{ .Release.Namespace }}
+{{- end }}
+{{- end }}
+
```

Comment:
Did you verify that this works? I'm not sure the rbac proxy (used for access to metrics endpoint) works without a cluster role. One thing we can do for now is to introduce a separate flag for enabling/disabling the rbac proxy and just keep if disabled by default.

Reply:
I will add a separate flag, in some environments this option should be omitted.

Reply:
I think that we can reuse the Prometheus flag, I don't think the rbac proxy is used for anything else (although I might be wrong). Yes, it makes sense to me to require both. If you have a counter suggestion please advise, I just want to be sure that whatever we do actually works.

Reply:
I remove the role/rolebinding for the rbac proxy; and only enable it if clusterscoped and manager.prometheus  are both true.